docs/source/user-guide/general/drift_guide.rst

```diff
@@ -0,0 +1,292 @@
+.. _drift_user_guide:
+
+====================
+Drift User Guide
+====================
+
+**In This Guide, You Can Learn:**
+
+* `What Is Distribution Drift and How Can You Detect It? <#what-is-distribution-drift>`__
+* `How Can You Use Deepchecks to Detect Drift? <#how-can-i-use-deepchecks-to-detect-drift>`__
+
+
+What Is Distribution Drift?
+==========================
+
+Distribution drift is a change in the underlying distribution of the data, the label or of the relationship between them. Drift happens over time,
+and can occur either gradually or suddenly.
+
+Common reasons for drift can be natural changes in the data, such as shopping habits changing for an online retailer
+(for example, during the covid pandemic, virtual shopping has risen significantly and buyer patterns have changed), or
+data acquisition and processing issues, such as a camera settings changing accidentally to have lower exposure.
+
+So Why Is Drift So Important?
+-----------------------------
+
+Machine learning models are meant to predict on unseen data, based on previous known data (more accurately, this refers
+to supervised machine learning models, for which drift is most relevant).
```

Comment:
Because we mainly talk about labels and the relationship to them, I thought this is worth mentioning. This is why it says "most relevant". Do you suggest we remove the parenthesis altogether? Or maybe soften it more?

Reply:
I suggest removing it, don't feel that it adds important details.
Ddon't really relate to your first part (the drift shouldn't be discussed mainly in relationship to the labels, also didn't really get a strong feeling that that is the case - for example - data drift, prediction drift...).